meta-mender-variscite/recipes-bsp/u-boot/u-boot-variscite.bbappend

```diff
@@ -17,3 +17,6 @@ do_deploy:append_var-som-mx6() {
 do_deploy:append_imx6ul-var-dart() {
     make_u_boot_spl_image
 }
+
+# For imx8mm-var-dart
+MENDER_UBOOT_AUTO_CONFIGURE = "0"
```

Comment:
see above, misses machine override

Reply:
Added in commit 89d22d4a4d6125d4b55b05624735c9b73db84e78